packages/__cxstarter__/src/components/Page/tokens/__cxstarter__Page.ts

```diff
@@ -12,11 +12,18 @@
  * limitations under the License.
  */
 
+import { on } from '@bodiless/fclasses';
 import { asElementToken } from '@bodiless/cx-elements';
-import { cxPage } from '@bodiless/cx-templates';
+import { cxPage, GenericTemplateClean } from '@bodiless/cx-templates';
+// import { cxPage } from '@bodiless/cx-templates';
+import { __cxstarter__GenericTemplate } from '../../GenericTemplate';
 
 const Default = asElementToken({
   ...cxPage.Default,
+  Components: {
+    // @todo fix Type error
+    _default: on(GenericTemplateClean)(__cxstarter__GenericTemplate.StyleGuide),
```

Comment:
@wodenx  how to address this type error:

src/components/Page/tokens/__cxstarter__Page.ts(25,5): error TS2322: Type '{ _default: HOCBase<any, any, any>; }' is not assignable to type 'Design<{}, any>'.

(similar error in Richtext tokens but its commented out for now.

Reply:
Try using asFluidToken instead of asElementToken -- asElementToken only allows `_` as the design key.